development/kops/run_sonobuoy.sh

```diff
@@ -26,7 +26,15 @@ if [[ "${KOPS_STATE_STORE}" != s3://* ]]
 then
   export KOPS_STATE_STORE="s3://${KOPS_STATE_STORE}"
 fi
-export KOPS_CLUSTER_NAME=$(kops get cluster --state "${KOPS_STATE_STORE}" | tail -n +2 | cut -f1 -d '  ' 2>/dev/null)
+if [ -z "${KOPS_CLUSTER_NAME}" ]
+then
+  export KOPS_CLUSTER_NAME=$(kops get cluster --state "${KOPS_STATE_STORE}" | tail -n +2 | cut -f1 -d '	' 2>/dev/null)
```

Comment:
were the extra spaces after the `-d` intentional here?

Reply:
That is actually a tab I think, cut only allows one character for -d as I recall